### ISSUE TITLE
Are you sure this code is running. There are a lot of miskates

### ISSUE THREAD

**Issue by jpainam** [OasisYang/SSG#18]
Hi; thank you for sharing
But your code has a lot of mistake; I'm inclined to say; it can't execute. Except we modified them. Pls, make sure you upload a running code out of the box?
For example; here; you used `cross_entropy` without `self.cross_entropy`. 
https://github.com/OasisYang/SSG/blob/b607ab650d2e01a984861a041b7cd248e727d621/reid/loss/weight_cross_entropy.py#L21

in reid/loss/triplet.py L35, the code is unreachable, you used `if False`

**Comment by jpainam** [comment]
Hi;  I suggest you try to solve these issues instead of just closing them. We understand it's your code and we would like to get similar results. This line of code I highlighted doesn't even compile because you wrote cross_entropy instead of F.cross_entropy or self.cross_entropy

Thank you for solving the issue to make your code clean for the community
